index.js

```diff
@@ -135,50 +177,63 @@ function nowServing(line) {
   return `Currently serving ${line.shift()}.`
 }
 
-const line = []
+function takeATicketNumber(line) {
+  const line_number = line.push((line.length) + 1)
```

Comment:
Rather than pushing the `length` of the line into the `line`, I would push the `ticketCounter` variable in to the line.
So, if someone's the 100th customer, but the line is empty, we push their number into the array rather than 1